specification/glossary.md

```diff
@@ -2,10 +2,130 @@
 
 *This section is non-normative.*
 
-`TODO` Needs to be completed and linked
+## Adjustment
 
-Resource:
-> Add definition
+A charge representing a modification to billing data to account for certain events or circumstances not previously captured, or captured incorrectly. Examples include: billing errors, service disruptions, or pricing changes.
 
-Service:
-> Add definition
+## Amortization
+
+The gradual recognition of upfront costs over the duration of time that the good or service is consumed. An upfront payment made toward a commitment based discount can be amortized over commitment life. Amortization is valuable when the commitment period (time duration of the cost) extends beyond the granularity of the source report.
+
+## Billed Cost
+
+The total amount of fees and discounts as documented in an itemized bill.
+
+## Billing Account
+
+A mandatory account established with a service provider to manage and pay for all charges associated with an invoice. A billing account might have sub-accounts, all of whose costs are consolidated to the billing account.
+
+## Billing Currency
+
+The specific currency in which invoices or bills for services are issued and payments are expected to be made.
+
+## Billing Period
+
+The time interval over which a customer's usage of services is measured for billing purposes. For most cloud providers, the billing period is a month.
+
+## Block Pricing
+
+Pricing approach where the cost of a particular resource or service is determined based on predefined quantities or tiers of usage. In these scenarios, the Pricing Quantity can be different from the Usage Quantity.
+
+## Charge
+
+A row in a FOCUS compatible cost and usage data associated with the usage of a service.
+
+## Commitment
+
+A customer's agreement to consume a specific quantity of a service or resource over a defined period, usually also creating a financial commitment throughout the entirety of the commitment period. Some commitments also hold Providers to certain assurance levels of resource availability.
+
+## Commitment-Based Discount
+
+Also known as Commitment Discount, this is a pricing strategy or incentive offered by a service provider to encourage customers to commit to a certain level of usage or purchase over a specified period. Offerings vary in specificity of commitment, penalties for underutilization, and possible remedies.
+
+## CSP
+
+Cloud Service Provider (e.g. Amazon Web Services, Microsoft Azure, Google Cloud Platform).
+
+## Dimension
+
+Specification-defined categorical attribute that provides context or categorization to billing data.
+
+## Effective Cost
+
+A comprehensive and realistic measure of the true cost of a service, inclusive of all reduced rates, discounts, and impacts of amortization.
+
+## Finalized Tag
+
+A tag with one tag value chosen from a set of possible tag values after being processed by a set of provider-defined or user-defined rules.
```

Comment:
We need to decide whether and how we'll be linking to column definitions.  I would love to see something like "for more, see the [Finalized Tags](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/blob/a18d86c50299c02a6237b23203b7481dd474af7b/specification/dimensions/tags.md#finalized-tags) section of the [Tags](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/blob/a18d86c50299c02a6237b23203b7481dd474af7b/specification/dimensions/tags.md) column definition.  That would be helpful context for me to know more about this term, which yields nothing upon Googling.

Reply:
@cnharris10 was the plan to move the content we put in the column spec here and link to here?

Reply:
We've gone back and forth on this.  We've settled on disclosing terms in the spec and linking to the glossary.  I like Shawn's suggestion but need buy-in.

Reply:
I'm going to leave as is and adjust potentially in the preview phase.